### ISSUE TITLE
Issue on Ubuntu 16.10

### ISSUE THREAD

**Issue by slemens** [SpaceK33z/plex2netflix#25]
I've just installed a new virtual machine with the Ubuntu 16.10 ISO image. After installing nodejs and git, I've tried to start plex2netflix, but I get a error message:

```
sudo plex2netflix --host 192.168.25.10 --token=qByeU187P1D7pjnXYix3
/usr/local/lib/node_modules/plex2netflix/lib/index.js:19
    let result = Promise.resolve();
    ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/plex2netflix/lib/cli.js:2:22)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

Can someboy help me fixing this issue?

Best regards,
Sebastian

**Comment by tdharris** [comment]
I also have this issue on Ubuntu 14.04.4 LTS

**Comment by SpaceK33z** [comment]
Yeah it works only on Node v6 at the moment. I kinda forgot about v4. I'll work on it soon.

**Comment by slemens** [comment]
Ok, that was a easy workaround, thanks for the hint ;).
For all other afftected Users, just use the Node.js repo, with the following commands:

```
curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
sudo apt-get install -y nodejs
```

Got it successfully to work.

**Comment by SpaceK33z** [comment]
This is fixed in `0.5.1`. Thanks for your bugreport.
